pyccl/tracers.py

```diff
@@ -616,9 +637,13 @@ class WeakLensingTracer(Tracer):
             normalization. Set to False to use the raw input amplitude,
             which will usually be 1 for use with PT IA modeling.
             Defaults to True.
+        n_samples (int, optional): number of samples over which the kernel
+            is desired. These will be equi-spaced in radial distance.
```

Comment:
Specify this is for the lensing kernel (i.e. this is not used for the IA contribution)